site/_includes/masthead.html

```diff
@@ -0,0 +1,21 @@
+{% assign site_logo = site.logo | relative_url %}
```

Comment:
apparently [GH Pages uses jekyll 3.9.3](https://pages.github.com/versions/) and `relative_url` is only available after jekyll 4.0.0.

i haven't looked at the GH Actions closely enough to know whether this would break on deploy.

Reply:
actually, nevermind. the change that I found in the 4.0.0-beta release post [only applies to `link`](https://github.com/jekyll/jekyll/pull/6727) as in `{% link "about.md" %}`

Reply:
Got it!  This is based on the theme, so relative_url definitely works here.